amplify/backend/api/themeetinghouse/schema.graphql

```diff
@@ -684,6 +684,31 @@ type Livestream
   titles: [String]
   homepageLink: String
 }
+
+type Announcement
+ @model
+ @auth(rules:[
+   {allow:public, operations:[read]},
+   {allow:private, operations:[read],provider:iam},
+   {allow: groups, operations:[read,update,delete,create],groups: ["Admin"],provider:userPools}
+ ])
+  @key(
+    name: "byParishByDate", 
+    fields: ["crossRegional", "parish", "expirationDate", "publishedDate"],
+    queryField: "listByParishByDates"
```

Comment:
Perhaps make this more descriptive, like `listAnnoucementByParishByDate`, otherwise we don't know what the query returns.

Reply:
Done. Thanks